2018-email

```diff
@@ -26,31 +36,37 @@ The second most important thing FOOD:
  booking the stay - or you are knowingly doing something else (like cooking yourselves).
 
 How to arrive to the venue?
- - a bus from Ivalo airport (on all flights, waits for everyone to arrive)
- - add time & date to Trello - know who to meet at the bus, or at the venue
- - meet on Sat / Sun on dinner. Sun after dinner we could open the space (if all have arrived).
+ - a bus is driving from Ivalo airport to Suomen Latu Kiilopää (on all flights, waits for everyone to arrive)
+ - add your time & date to Trello - know who to meet at the bus, or at the venue
+ - depending on when most people arrive we well open the space, but the venue is so small that we will
+   easily bump into each other.
 
 What to expect?
- - open space, market place in Trello.
- - outdoor activities during day time, conferencing later.
+ - CodeFreeze is an Unconference which is nearly entirely run by you, the participants.
+   Typically there are 3-4 days of conference mixed with a lot of outdoor activities.
+ - We will start with a marketplace where everyone can introduce themselfes and share the
```

Comment:
```suggestion
 - We will start with a marketplace where everyone can introduce themselves and share the
```

Reply:
Thanks! 
But aren't we all a little bit Elf? ;-)